.babelrc

```diff
@@ -1,4 +1,12 @@
 {
-  "presets": [["es2015", { "modules": false }], "react"],
-  "plugins": ["transform-class-properties"]
-}
+	"env": {
+		"production": {
+			"presets": [["es2015", { "modules": false }], "react"],
+			"plugins": ["transform-class-properties"]
```

Comment:
`plugins` can be moved outside because they apply to both environments.

```json
{
        "plugins": ["transform-class-properties"],
	"env": {
		"production": {
			"presets": [["es2015", { "modules": false }], "react"]
		},
		"development": {
			"presets": [["es2015", { "modules": false }], "react", "react-hmre"]
		}
	},
}
```

Also is this an 8 spc tab?